src/quarkus-cheat-sheet/quarkus-cheat-sheet.html

```diff
@@ -6615,7 +6615,7 @@ <h3>Authors :</h3>
 </div>
 
 <div class="author-bio">0.26.1</div>
-<img src="./redhat/assets/by-nc-sa.png"/>
+<img src="/home/guillaume/workspace/opensource/temp/quarkus-cheat-sheet/src/quarkus-cheat-sheet/template/assets/by-nc-sa.png"/>
```

Comment:
Static file to your computer.

Reply:
I'm using `{__dirname}` in the `template.js` to resolve the absolute path of the script:

```js
<link href="${__dirname}/assets/style.css" rel="stylesheet">
```

So the path is actually relative to the `template.js` file. 
I don't think we should versionned this "temporary" file. It's useful for debugging purpose but it's really a "transient state".

If you want to keep the HTML files versionned I can revert this change even though we might run into troubles if the HTML file is generated somewhere else.